nixops_proxmox/backends/proxmox.py

```diff
@@ -381,7 +381,7 @@ def _configure_initial_nix(self, uefi: bool, instance_id: Optional[int] = None):
             }
         else:
             # Use nix2py to read self.fs_info.
```

Comment:
@RaitoBezarius I'm not sure I understand the comment here. Is this related to the future plans for a `nixpart` integration?

Reply:
Yes, indeed, it was supposed to be used for a more Nixy way to partition.